boards/arm/nrf52840dk_nrf52840/Kconfig

```diff
@@ -0,0 +1,5 @@
+config BOARD_ENABLE_DCDC
```

Comment:
??

There already is an nrf52840dk_nrf52840 board definition upstream; why is it being copied here?

Maybe rename it or something like that, and add a README explaining what's going on here with a pointer to the board porting guide?

Reply:
renamed to `custom_nrf52840dk` to emphasize it's a "custom board".